optimum/utils/input_generators.py

```diff
@@ -53,6 +53,7 @@ def wrapper(*args, **kwargs):
     "width": 64,
     "height": 64,
     "num_channels": 3,
+    "visual_seq_length": 5,
     # audio
```

Comment:
apparently `num_of_detection_patches` is missing here according to some non-passing tests